src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/QueryIT.java

```diff
@@ -99,6 +99,49 @@ public void multipleFromTest() throws IOException {
         Assert.assertEquals(14, getTotalHits(response));
     }
 
+    @Test
+    public void selectAllAndSpecificFieldTest() throws IOException {
+        String[] arr = new String[] {"account_number", "firstname", "address", "birthdate", "gender", "city", "lastname", "balance", "employer", "state", "age", "email", "male"};
+        Set<String> expectedSource = new HashSet<>(Arrays.asList(arr));
+        JSONObject response = executeQuery(String.format(Locale.ROOT,
+                "SELECT *, age FROM %s/account LIMIT 5", TestsConstants.TEST_INDEX_BANK));
+        Assert.assertTrue(response.has("hits"));
+        JSONArray hits = getHits(response);
+        for (int i = 0; i < hits.length(); ++i) {
+            JSONObject hit = hits.getJSONObject(i);
+            Assert.assertEquals(expectedSource, getSource(hit).keySet());
+        }
+    }
+
+    @Test
+    public void selectAllAndSpecificFieldwithGroupByTest() throws IOException {
+        String[] arr = new String[] {"account_number", "firstname", "address", "birthdate", "gender", "city", "lastname", "balance", "employer", "state", "age", "email", "male"};
+        Set<String> expectedSource = new HashSet<>(Arrays.asList(arr));
+        JSONObject response = executeQuery(String.format(Locale.ROOT,
+                "SELECT *, age FROM %s/account GROUP BY age LIMIT 5", TestsConstants.TEST_INDEX_BANK));
+        Assert.assertTrue(response.has("hits"));
+        JSONArray hits = getHits(response);
```

Comment:
remove copypaste from tests (that hits check seem to be repeated between all tests introduced).

Please look into SubqueryIT.java for better approach of validation of the query results.